dotcom-rendering/src/web/components/FixedMediumSlowXIIMPU.tsx

```diff
@@ -227,17 +228,6 @@ const Card33_Card33_Ad33 = ({
 	</UL>
 );
 
-/**
- *
- * When there is an odd number of cards the last card spans
- * both columns so we don't want to push the divider for the
- * previous two cards above it down. If we did it would
- * touch
- */
-function shouldOffsetWhenOdd(i: number, noOfCards: number) {
-	return i === noOfCards - 2 || i === noOfCards - 3;
-}
-
```

Comment:
This is not a consistent abstraction, so it has been turned into a helper in `dynamicSlices`.